### PR TITLE
Bt device

### PR DESCRIPTION
**Pull request by natdan**

### CODE REVIEW

File: bt_device_classes.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+
+# Major Service Class
+
+LIMITED_DISCOVERABLE_MODE = 0x2000
+POSITIONING = 0x10000
+NETFORKING = 0x20000
```
Comment (pdenes) [inline bt_device_classes.py]: Ha, found another typo! :)

File: bluetooth_gamepad.py [hunk 2]
```diff
@@ -21,81 +21,75 @@
 from usb_hid_report_descriptor import UsagePage, Usage, Collection, GenericDesktopCtrls, Var, Abs, NoWrap, Linear, PreferredState, NoNullPosition, \
```
Comment (pdenes) [inline bluetooth_gamepad.py:21]: Not related to these changes, just noticed that _some_ classes are imported here from usb_hid_report_descriptor, but not all, so some are used as "usb_hid_report_descriptor.[thing]", some are as "[thing]" below - really just a minor cosmetic thing...

**Review by pdenes — COMMENTED** [review]
Looks good! :)

File: bt_device_classes.py (hunk 1 above)
Comment (natdan) [reply bt_device_classes.py]: Well spotted.

File: bluetooth_gamepad.py (hunk 2 above)
Comment (natdan) [reply bluetooth_gamepad.py:21]: I completely understand what you're saying but not 100% sure how to fix it. There are so many things that have 'generic' names - Gamepad for instance - both sdp_record and usb_hid_report_descriptor - currently they are with different cases but in next PR I'll fix that, too; Application, Data or Report on the other hand are way to generic and it is so easy to fall in trap mixing several imports with the same name - and with Python you get the see only the latest... 
This is something to keep an eye on and try to make more uniform/less ambiguous...

**Pull request merged**